lib/jmap/core/response/response_object.dart

```diff
@@ -24,9 +24,11 @@ class ResponseObject with EquatableMixin {
 
   Map<String, dynamic> toJson() => _$ResponseObjectToJson(this);
 
-  T? parse<T extends MethodResponse>(MethodCallId methodCallId, T fromJson(Map<String, dynamic> o)) {
+  T? parse<T extends MethodResponse>(MethodCallId methodCallId, T fromJson(Map<String, dynamic> o), {MethodName? methodName}) {
```

Comment:
why we need `methodName` to parse?

Reply:
Because `response` returns when `request` has both `EmailSubmission/Set` & `Email/Set` with 2 `methodName` and one `methodCallId`

Reply:
really?

Reply:
You can watch the video here or you can try again with your postman. Don't ask such stupid questions.

https://user-images.githubusercontent.com/80730648/132333448-725a023b-65e9-44f5-854e-45a1a4c8dec6.mov

Reply:
:) I like your video

Reply:
@chibenwa can you explain to me why 2 methods but have 3 responses